signatures/rego/syscall_table_hooking_test.rego

```diff
@@ -0,0 +1,40 @@
+package tracee.TRC_15
+
+test_match_1 {
+    tracee_match with input as {
+        "eventName": "detect_hooked_syscalls",
+        "argsNum": 1,
+        "args": [
+            {
+                "name": "hooked_syscalls",
+                "value": [{"SyscallName":"kill","ModuleOwner":"diamorphine"},{"SyscallName":"getdents","ModuleOwner":"diamorphine"},{"SyscallName":"getdents64","ModuleOwner":"diamorphine"}]
+            }
+        ]
+    }
+}
+
+test_match_2 {

```

Comment:
What is the difference between test_match_1 and test_match_2?
You need to have unique tests, there is no real use to doing the same test twice.
You can test for example the behavior with 1 hooked syscall and multiple hooked syscalls as different senerios.